lab_1/main.py

```diff
@@ -1,94 +1,121 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import math
 
 
+def checker(con_object, type_ob):
+    if type_ob is list and type(con_object) is list:
+        len_ = len(con_object) > 0
```

Comment:
It is usually considered a bad style to use built-ins (e.g. `len`, `str`, `list`) as variable names.

Reply:
Okay, changed.